docs/en/ingest-management/elastic-agent/uninstall-elastic-agent.asciidoc

```diff
@@ -22,17 +22,39 @@ stops and uninstalls any managed programs, such as {beats} and
 
 If you run into problems, see <<fleet-troubleshooting>>.
 
+For DEB or RPM installations, you must uninstall the {agent} manually.
+
+//QUESTION: Is this true ^^? Are there any other ways to uninstall that are less
+//manual?
+
 [discrete]
-== Uninstall on DEB or RPM
+== Uninstall manually
+
+//QUESTION: Are the following steps complete and accurate? 
+
+You might need to uninstall {agent} manually if there's a failure during
+installation, or you're using a distribution, like DEB or RPM, that doesn't
+support the `uninstall` command.
+
+To uninstall {agent} manually and completely remove it from your system:
+
+. <<unenroll-elastic-agent,Unenroll the agent>> if it's managed by {fleet}.
+
+. For standalone agents, back up any configuration files you want to preserve. 
 
-The `uninstall` command is not supported for DEB or RPM installations. To
-uninstall {agent} on DEB or RPM:
+. On your host, <<stop-elastic-agent,stop the agent>>. If any {agent}-related
+processes are still running, stop them too.
++
+TIP: Search for these processes and stop them if they're still running:
+`filebeat`, `metricbeat`, `fleet-server`, and `elastic-endpoint`.
 
-. If the agent is managed by {fleet}, <<unenroll-elastic-agent,unenroll the agent>>.
-. On your host, <<stop-elastic-agent,stop the agent>>.
-. Manually remove the agent files from your system.
+. Manually remove the {agent} files from your system. For example, if you're
+running {agent} on macOS, delete `/Library/Elastic/Agent/*`. Not sure where the
+files are installed? See <<installation-layout>>.
 
-//TODO: Confirm this procedure.
+. If you've configured the Elastic Endpoint integration, also remove the files
+installed for endpoint protection. The directory structure is similar to {agent},
+for example, `/Library/Elastic/Endpoint/*`.
 
```

Comment:
this is where I would link to the Endpoint docs, with a brief note akin to the below.  I also added a note about removing the RPM/Deb packages, too.
```suggestion

. When Elastic Endpoint integration is removed on a macOS host the Endpoint System Extension is left on disk intentionally, if you wish to remove it, please see the Elastic Endpoint documentation. 

. If you are using DEB or RPM, you can now use the package manager to remove the installed package.
```

Reply:
Makes sense. I will implement this change in the source files because I need to fix formatting and add attributes.